content/altinn-studio/guides/options/_index.nb.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Kodelister (options)
+title: Kodelister (svaralternativer)
 linktitle: Kodelister
-description: Hvordan konfigurere options/kodelister for en app?
+description: Hvordan konfigurere svaralternativer/kodelister for en app?
 toc: true
 weight: 40
 ---
 
-Altinn tilbyr to ulike måter en app kan eksponere kodelister - statisk og dynamisk. Disse eksponeres primært fra options endepunktet i appen, og er tilgjengelig på `{org}/{app}/api/options/{optionsId}`.
-Checkbox, Dropdown og RadioButton komponenter vil automatisk kunne hente ut en slik liste om man kobler denne komponenten til en slik options-id. Men ikke alle dynamiske kodelister må gå via api'et - vi har også dynamiske kodelister som baserer seg på verdiene fra en repeterende struktur i datamodellen.
+Altinn tilbyr to ulike måter en app kan eksponere kodelister på: Statisk og dynamisk. Disse eksponeres primært fra endepunktet som er tilgjengelig på `{org}/{app}/api/options/{optionsId}`, hvor `optionsId` er ID-en til listen.
```

Comment:
Som ikke-teknisk bruker forstår jeg her verken forskjellen på statisk og dynamisk, eller hva det betyr at den blir eksponert. Har du mulighet for å skrive det tydeligere (bruke andre ord), med tanke på en målgruppe som ikke er teknisk?